portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.path

```diff
@@ -0,0 +1,56 @@
+<html>
+<head>
+<title>MyTestFile</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">MyTestFile</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>OPEN_SIMULATION</td>
+	<td>//button[@data-qa-id='simulation']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CUSTOM_SCREEN</td>
+	<td>//span[text()='Custom']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CUSTOM_HEIGHT</td>
+	<td>//input[@id='_com_liferay_product_navigation_simulation_web_portlet_SimulationPortlet_height']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CUSTOM_WIDTH</td>
+	<td>//input[@id='_com_liferay_product_navigation_simulation_web_portlet_SimulationPortlet_width']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APPLY_CUSTOM_SIZE</td>
+	<td>//button[@class='w-100 btn btn-secondary']</td>
```

Comment:
Essas classes sao um pouco genericas, o que nao eh um problema em si se esse for o objetivo da path. Mas seria bom usar o `contains()` pra usar somente uma classe.